cluster/gce/ubuntu/master.yaml

```diff
@@ -1,128 +0,0 @@
-      ExecStartPre=/bin/chmod 544 /home/kubernetes/bin/configure.sh
-      ExecStart=/home/kubernetes/bin/configure.sh
-
-      [Install]
-      WantedBy=kubernetes.target
-
-  - path: /etc/systemd/system/kube-master-configuration.service
-    permissions: 0644
-    owner: root
-    content: |
-      [Unit]
-      Description=Configure kubernetes master
-      After=kube-master-installation.service
-
-      [Service]
-      Type=oneshot
-      RemainAfterExit=yes
-      ExecStartPre=/bin/chmod 544 /home/kubernetes/bin/configure-helper.sh
-      ExecStart=/home/kubernetes/bin/configure-helper.sh
-
-      [Install]
-      WantedBy=kubernetes.target
-
-  - path: /etc/systemd/system/kube-docker-monitor.service
-    permissions: 0644
-    owner: root
-    content: |
-      [Unit]
-      Description=Kubernetes health monitoring for docker
-      After=kube-master-configuration.service
-
-      [Service]
-      Restart=always
-      RestartSec=10
-      RemainAfterExit=yes
-      RemainAfterExit=yes
-      ExecStartPre=/bin/chmod 544 /home/kubernetes/bin/health-monitor.sh
-      ExecStart=/home/kubernetes/bin/health-monitor.sh docker
-
-      [Install]
-      WantedBy=kubernetes.target
-
-  - path: /etc/systemd/system/kubelet-monitor.service
-    permissions: 0644
-    owner: root
-    content: |
-      [Unit]
-      Description=Kubernetes health monitoring for kubelet
-      After=kube-master-configuration.service
-
-      [Service]
-      Restart=always
-      RestartSec=10
-      RemainAfterExit=yes
-      RemainAfterExit=yes
-      ExecStartPre=/bin/chmod 544 /home/kubernetes/bin/health-monitor.sh
-      ExecStart=/home/kubernetes/bin/health-monitor.sh kubelet
-
-      [Install]
-      WantedBy=kubernetes.target
-
-  - path: /etc/systemd/system/kube-logrotate.timer
-    permissions: 0644
-    owner: root
-    content: |
-      [Unit]
-      Description=Hourly kube-logrotate invocation
-
-      [Timer]
-      OnCalendar=hourly
-
-      [Install]
-      WantedBy=kubernetes.target
-
-  - path: /etc/systemd/system/kube-logrotate.service
-    permissions: 0644
-    owner: root
-    content: |
-      [Unit]
-      Description=Kubernetes log rotation
-      After=kube-master-configuration.service
-
-      [Service]
-      Type=oneshot
-      ExecStart=-/usr/sbin/logrotate /etc/logrotate.conf
-
-      [Install]
-      WantedBy=kubernetes.target
-
-  - path: /etc/systemd/system/kubernetes.target
-    permissions: 0644
-    owner: root
-    content: |
-      [Unit]
-      Description=Kubernetes
-
-      [Install]
-      WantedBy=multi-user.target
-      After=docker.service
-
-runcmd:
- - systemctl daemon-reload
- - systemctl enable kube-master-installation.service
- - systemctl enable kube-master-configuration.service
- - systemctl enable kube-docker-monitor.service
- - systemctl enable kubelet-monitor.service
- - systemctl enable kube-logrotate.timer
- - systemctl enable kube-logrotate.service
- - systemctl enable kubernetes.target
```

Comment:
This is needed by Ubuntu.

This extra "enable" call ensures that the appropriate symlink is created by systemd (so that it is brought back up after a reboot).